### PR TITLE
frontend: add poster attribute video tag

### PR DESCRIPTION
**Pull request by NicolaLS**
In the scan-qr-video component, we use a html video tag which has an unappealing placeholder image. Adding a poster attribute changes that placeholder image, we use an empty svg in this case.

### CODE REVIEW

**Comment by NicolaLS** [comment]
this is how it looks without the attribute:
Screenshot_20231012_203128_BitBoxApp DEBUG

and this with the `poster` tag containing an empty svg (the animation of the three dots make it look weird on that screenshot):

Screenshot_20231012_204944_BitBoxApp DEBUG

File: frontends/web/src/routes/account/send/components/inputs/scan-qr-video.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const ScanQRVideo = ({
       <video
         className={style.qrVideo}
         ref={videoRef}
+        poster="%3Csvg%20xmlns=%22http://www.w3.org/2000/svg%22%20width=%2264%22%20height=%2248%22%3E%3C/svg%3E'"
```
Comment (thisconnect) [inline frontends/web/src/routes/account/send/components/inputs/scan-qr-video.tsx:47]: Please remove the singlequote at the end.

**Review by thisconnect — CHANGES_REQUESTED** [review]

**Review by thisconnect — APPROVED** [review]
untested LGTM 👍

**Review by Beerosagos — APPROVED** [review]
tACK on android and qt-linux! :+1:

**Pull request merged**